httpapi/httpapi.go

```diff
@@ -43,6 +43,7 @@ func NewRouter(hy *hydra.Hydra) *mux.Router {
 	mux.HandleFunc("/idgen/add", idgenAddHandler()).Methods("POST")
 	mux.HandleFunc("/idgen/remove", idgenRemoveHandler()).Methods("POST")
 	mux.HandleFunc("/swarm/peers", swarmPeersHandler(hy))
+	mux.HandleFunc("/pstore/list", pstoreListHandler(hy))
```

Comment:
Thought: we already have per-head output in `/heads`, what if we add this to `/heads` as a query arg, something like `/heads?peers=true`?

Reply:
Currently, that `/heads` endpoint returns

```text
{"ID":"...","Addrs":[...]}
{"ID":"...","Addrs":[...]}
... (for each head)
```

Would you return peerstore data _instead_ of the head information or somehow interleaved in the response?

I think I would opt against interleaving as it would make parsing more difficult and if we toggle the functionality of the endpoint with a query parameter I think I would also prefer an explicit/separate endpoint :)

Reply:
I was thinking just adding another field like 

```
{"ID":"...","Addrs":[...],"Peers":[{"AddrInfo":...,"AgentVersion":...,"Protocols":[...]},...]}
```

Reply:
(Seems tidier, but I don't feel too strongly about this either, so I'm good with either.)